src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -372,3 +372,18 @@ def validate_label(label):
                        "characters, '-', '_' or '.', and must start and end with an alphanumeric character" % label)
 
     return {kv[0]: kv[1]}
+
+
+def validate_max_surge(namespace):
+    """validates parameters like max surge are postive integers or percents. less strict than RP"""
+    if namespace.max_surge is None:
+        return
+    intOrPercent = namespace.max_surge
```

Comment:
`intOrPercent` is not pythonic, we can use `int_or_percent` instead.

Reply:
Yep on it

Reply:
done.